backend/tournesol/serializers/rating.py

```diff
@@ -40,6 +43,21 @@ def get_n_comparisons(self, obj):
             & (Q(entity_1=obj.entity) | Q(entity_2=obj.entity))
         ).count()
 
+    @extend_schema_field(OpenApiTypes.DATETIME)
+    def get_comparison_date(self, obj):
+        """
+        The dates of each comparisons
+        """
+        if hasattr(obj, "comparison_date"):
+            # Use annotated field if it has been defined by the queryset
+            return obj.comparison_date
+        
+        comparisons = obj.user.comparisons.filter(
+            Q(poll=self.context["poll"])
+            & (Q(entity_1=obj.entity) | Q(entity_2=obj.entity))
+        )
+        return list(comparisons.values_list("datetime_add", flat=True))
```

Comment:
This should not return a list. This method is model specific, and thus will always return one item.

If we check what the API return at `localhost:8000/docs/#/users/users_me_contributor_ratings_list/`, we currently have:

```json
      "comparison_date": [
        "2022-10-17T15:43:16.201273Z"
      ]
```

We should have

```
      "comparison_date": "2022-10-17T15:43:16.201273Z"
```

I'll fix.

Reply:
I realized there is no field `last_compared_at` in the `ContributorRating` model.

If we want the ratings to be ordered by the last compared date, we need to fetch the `datetime_add` value directly in the `tournesol.views.ratings.get_annotated_ratings` function.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1170/commits/e55a9b88fed1e41a852567b871e2d3ee22da3084.

Reply:
Thank you for your review! I see you are currently working on some changes, is there anything I can do?

Reply:
My pleasure @lk067fn 

I'm currently updating the tests. I noticed all ratings have the same `last_compared_at` value, that's why they always pass :sweat_smile: I fix this I'll just specify different comparison date in the `setUp` function.

> is there anything I can do?

I think you are good with this PR : ) thank you very much. I'll merge as soon as I reviewed everything.